app/api/dao/organization.py

```diff
@@ -88,6 +70,89 @@ def update_organization(data):
             return update(organization, data, messages.ORGANIZATION_SUCCESSFULLY_UPDATED, HTTPStatus.OK)     
         return messages.NOT_ORGANIZATION_REPRESENTATIVE, HTTPStatus.FORBIDDEN
 
+    
+    @staticmethod
+    def list_organizations(
+        search_query,
```

Comment:
rename this to organization_name

Reply:
I must've missed that. Done. It's now renamed to search_name